tests_common/test_utils/api_fastapi.py

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import json
+
+from airflow.models import Log
+from airflow.sdk.execution_time.secrets_masker import DEFAULT_SENSITIVE_FIELDS as sensitive_fields
+
+
+def _masked_value_check(data, sensitive_fields):
+    """
+    Recursively check if sensitive fields are properly masked.
+
+    :param data: JSON object (dict, list, or value)
+    :param sensitive_fields: Set of sensitive field names
+    """
+    if isinstance(data, dict):
+        for key, value in data.items():
+            if key in sensitive_fields:
+                assert value == "***", f"Expected masked value for {key}, but got {value}"
```

Comment:
Do we actually want to print the value here?

Reply:
That code is only used in testing, I think it's ok to print there.

Reply:
I did not read the module name properly. Thanks!